dataland-backend/Dockerfile

```diff
@@ -1,10 +1,7 @@
 FROM eclipse-temurin:17.0.2_8-jre-alpine
 
-COPY gradle /app/gradle/
-COPY dataland-backend/src /app/dataland-backend/src/
-COPY dataland-backend/build.gradle.kts /app/dataland-backend
-COPY gradlew settings.gradle.kts build.gradle.kts /app/
+COPY . /app/

```

Comment:
Wenn alles kopiert wird sind auch viele nicht verwendete Dinge dabei, warum den Container nicht so schlank wie möglich halten? Könnte man hier nicht sogar nur ein jar reinnehmen und ausführen?

Reply:
Ja - das ist blöde. 
Grund dafür: Das gradle-git-properties erwartet, dass es innerhalb eines git-repos ausgeführt wird - sonst failt der gradle befehlt.
Ausserdem: Das macht das Dockerfile schlanker. 
Da das resultierende Image nur im Test lebt und nicht irgendwoanders hingelangt finde ich es hier in Ordnung. 

Was denkst du?

Reply:
Ja wie gestern besprochen können wir es so lassen.